### PR TITLE
Data models for tags , items and levels

### PR DESCRIPTION
**Pull request by heyrandhir**
Updated the data models for 
- [x] tags
- [x]  items
- [x]  levels

### CODE REVIEW

File: tags/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# tags firestore collection data model
+
+```
+{
+  'id': string,
+  'date': {
+    "_seconds": number,
```
Comment (DashDeipayan) [inline tags/README.md:7]: shouldn't this be a ```timestamp```

**Review by DashDeipayan — CHANGES_REQUESTED** [review]

File: tags/README.md (hunk 1 above)
Comment (heyrandhir) [reply tags/README.md]: yeah correct. Fixed it.

Comment (DashDeipayan) [reply tags/README.md]: that's weird. Feels a bit in-consistent. Let me discuss this with Vinayak

Comment (heyrandhir) [reply tags/README.md]: Hi @DashDeipayan The database had some stale data, I created a fresh tag and it is storing as timestamp. 
image

**Review by DashDeipayan — APPROVED** [review]
LGTM

**Review by vinayak-trivedi — APPROVED** [review]
LGTM

File: items/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# items firestore collection data model
+
+```
+{
+  'id': string,
+  'itemId': string,
+  'itemType': string,
+  'levelId': string,
+  'levelName': string,
+  'levelNumber': number,
```
Comment (ankushdharkar) [inline items/README.md:10]: Can this be :

```
level: {
  id:
  name:
  value:
 }
 ```

**Review by ankushdharkar — CHANGES_REQUESTED** [review]

File: items/README.md (hunk 2 above)
Comment (heyrandhir) [reply items/README.md:10]: Created the refactor ticket here https://github.com/Real-Dev-Squad/website-backend/issues/974

**Comment by heyrandhir** [comment]
We need to modify the data model for items. the refactor ticket can be found [here](https://github.com/Real-Dev-Squad/website-backend/issues/974)

**Pull request merged**